client/apps/settings/index.coffee

```diff
@@ -0,0 +1,14 @@
+#
+# CRUD for curations
+#
+
+express = require 'express'
+routes = require './routes'
+
+app = module.exports = express()
+app.set 'views', __dirname
+app.set 'view engine', 'jade'
+
+app.get '/settings', routes.index
+app.get '/settings/:id/edit', routes.edit
+app.post '/settings/:id', routes.save
```

Comment:
this part felt a little wrong -- maybe it should be posting to curations? the route takes care of that but the naming is a little off.